File: Data Structures & Algorithms/pow-x-n/submission-1.java
class Solution {
    public double myPow(double x, int n) {
            double answer = 1;
        if(n == 0){
            return answer;
        }
        else if(n > 0){
            for(int z = 0; z < n; z++){
                answer = answer * x;
            }

        } else{
            int index = n*(-1);
            double denom = 1;
            for(int z = 0; z < index; z++){
                denom = denom * x;
            }
            answer = 1/denom;
            return answer;
        }
                return answer;

    }
}

File: Data Structures & Algorithms/daily-temperatures/submission-1.py
class Solution:
    def dailyTemperatures(self, temperatures: List[int]) -> List[int]:
        
        res = [0]*len(temperatures)
        stack = []

        for index, temp in enumerate(temperatures):
            while stack and temp > stack[-1][0]:
                tempV, tempInd = stack.pop()
                res[tempInd] = index - tempInd
            else:
                stack.append([temp,index])
        
        return res
            
